plugin-server/src/utils/db/sql.ts

```diff
@@ -110,4 +110,5 @@ export async function disablePlugin(hub: Hub, pluginConfigId: PluginConfigId): P
         [pluginConfigId],
         'disablePlugin'
     )
+    await hub.db.redisPublish(hub.PLUGINS_RELOAD_PUBSUB_CHANNEL, 'reload!')
```

Comment:
Q: Is there appropriate logging elsewhere in this path? If not, we should add it here.

Reply:
Added a metric to `processFatalVmSetupError`.